manual/thread_safetiness/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Thread safetiness
```

Comment:
```suggestion
title: Thread safety
```
(google claim safetiness is (for now?) a misspelling)

Reply:
Oh. But the noun would then be rather `thread safeness`?

Reply:
I have never seen "safeness" used in this  context (but google says it is used albeit "rarely")

Reply:
May some general remarks from my side.

I would like to include the content in one of the existing files/chapters, for example in https://root.cern/manual/root_classes_data_types_and_global_variables/
Otherwise the Basics part becomes too large. May we must reorganize or rewrite one of the existing chapters, but the aim should be not to let the first part become too overwhelming.

Avoid passive. In the ROOT Manual we talk to the reader directly.
Example: "...objects must not be used concurrently by different threads": Who is the actor? If the user is the actor, write: "Make sure that objects are not used concurrently by different threads".
If the application is the actor, name the actor and write: "xxx ensures that objects are not used concurrently by different threads"".


If a class is mentioned the first time, link to the appropriate page in the Reference Guide. Olivier creates a nice macro {% include ref class="ClassName" %}, for example {% include ref class="TDirectory" %} links to https://root.cern/doc/master/classTDirectory.html

Do not write "TODO" on the page that should be published. We do not indicate a work on progress in the Manual. By default a manual is always work in progress :-)

Reply:
fwiw I agree with @pcanal that the term is [thread safety](https://en.wikipedia.org/wiki/Thread_safety)

Reply:
@JohannesHentrich Thanks for the comments!  Should be all addressed expect for the place of this section.  I'll leave it to @couet to decide where to put it.